__tests__/__fixtures__/postman/petstore.collection.json

```diff
@@ -0,0 +1,2056 @@
+{
```

Comment:
I created this collection by importing [3.0/petstore.json](https://github.com/readmeio/oas-examples/blob/main/3.0/json/petstore.json) into Postman and then exporting it as a v2.1 collection.